DribbblishDynamic/dribbblish-dynamic.js

```diff
@@ -188,12 +188,10 @@ function getAlbumInfo(uri) {
 }
 
 function isLight(hex) {
-    var bigint = parseInt(hex.replace("#",""), 16);
```

Comment:
please convert strings to integers
`var [r,g,b] = hexToRgb(hex).split(',').map(Number);`

Reply:
done